src/test/java/org/yu55/yagga/handler/git/GitPullSchedulerTest.java

```diff
@@ -0,0 +1,27 @@
+package org.yu55.yagga.handler.git;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+public class GitPullSchedulerTest {
+
+    @Test
+    public void testPull() throws Exception {
+        //given
```

Comment:
TRIVIAL: we used to put space between slash and 'given/when/then' word: //given -> // given